generator/src/main/java/org/corfudb/generator/operations/NestedTxOperation.java

```diff
@@ -0,0 +1,66 @@
+package org.corfudb.generator.operations;
+
+import ch.qos.logback.classic.Logger;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'ch.qos.logback.classic.Logger'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)